Block/Adminhtml/System/Config/Fieldset/Payment.php

```diff
@@ -0,0 +1,99 @@
+<?php
+namespace Omise\Payment\Block\Adminhtml\System\Config\Fieldset;
+
+/**
+ * Fieldset renderer for Omise solution
+ */
+class Payment extends \Magento\Config\Block\System\Config\Form\Fieldset
+{
+    /**
+     * @param \Magento\Backend\Block\Context $context
+     * @param \Magento\Backend\Model\Auth\Session $authSession
+     * @param \Magento\Framework\View\Helper\Js $jsHelper
+     * @param \Magento\Config\Model\Config $backendConfig
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Backend\Block\Context $context,
+        \Magento\Backend\Model\Auth\Session $authSession,
+        \Magento\Framework\View\Helper\Js $jsHelper,
+        \Magento\Config\Model\Config $backendConfig,
+        array $data = []
+    ) {
+        $this->_backendConfig = $backendConfig;
```

Comment:
@jacekstan Do we need this line? Can't find anyplace that use this `$this->_backendConfig`